From 088be54e73a90539fc525ca96a3929b332561a90 Mon Sep 17 00:00:00 2001
From: Drew Crawford <drew@sealedabstract.com>
Date: Sat, 27 Feb 2016 03:10:18 -0600
Subject: [PATCH] Fix foundation build when libdispatch is enabled

David Grove writes:

>  If both --foundation and --libdispatch are given to the invocation of
>  build-script (or are in the same preset), then the build attempts to
>  build CoreFoundation with libdispatch dependent functionality
>  enabled. Right now, this doesn't compile, so the build fails.

Disable this behavior.
---
 build.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/build.py b/build.py
index d713b08..40753bf 100644
--- a/build.py
+++ b/build.py
@@ -61,13 +61,6 @@ if "XCTEST_BUILD_DIR" in Configuration.current.variables:
 		'-I/usr/include/libxml2'
 	]
 
-if "LIBDISPATCH_SOURCE_DIR" in Configuration.current.variables:
-        foundation.CFLAGS += " "+" ".join([
-                '-DDEPLOYMENT_ENABLE_LIBDISPATCH',
-                '-I'+Configuration.current.variables["LIBDISPATCH_SOURCE_DIR"],
-                '-I'+Configuration.current.variables["LIBDISPATCH_BUILD_DIR"]+'/tests'  # for include of dispatch/private.h in CF
-        ])
-
 foundation.SWIFTCFLAGS = " ".join(swift_cflags)
 
 foundation.LDFLAGS += '-lpthread -ldl -lm -lswiftCore -lxml2 '
-- 
2.5.4 (Apple Git-61)

